### PR TITLE
BROKER : Insert Address

### PR DESCRIPTION
**Pull request by GracelinJenifa**

### CODE REVIEW

File: AddressApp.ConsoleApp/Brokers/Storages/StorageBroker.Address.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+using System.Collections.Generic;
+using AddressApp.ConsoleApp.Models.Address;
+
+namespace AddressApp.ConsoleApp.Brokers.Storages
+{
+    partial class StorageBroker : IStorageBroker
+    {
+        List<Address> Addresses = new List<Address>();
+        public Address InsertAddress(Address Address)
```
Comment (ShriHumrudha) [inline AddressApp.ConsoleApp/Brokers/Storages/StorageBroker.Address.cs]: Correct the formatting here.

File: AddressApp.ConsoleApp/Models/Address/Address.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+namespace AddressApp.ConsoleApp.Models.Address
+{
+    internal class Address
+    {
+        public int Id { get; set; }
+        public string Name { get; set; }
+
```
Comment (ShriHumrudha) [inline AddressApp.ConsoleApp/Models/Address/Address.cs]: Remove the extra line.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: AddressApp.ConsoleApp/Brokers/Storages/StorageBroker.Address.cs (hunk 1 above)
Comment (GracelinJenifa) [reply AddressApp.ConsoleApp/Brokers/Storages/StorageBroker.Address.cs]: Please check I have corrected it.

File: AddressApp.ConsoleApp/Models/Address/Address.cs (hunk 2 above)
Comment (GracelinJenifa) [reply AddressApp.ConsoleApp/Models/Address/Address.cs]: Extra line removed.

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**